modules/templates/helper.go

```diff
@@ -976,7 +976,7 @@ type remoteAddress struct {
 
 func mirrorRemoteAddress(ctx context.Context, m *repo_model.Repository, remoteName string) remoteAddress {
 	a := remoteAddress{}
-	if !m.IsMirror {
+	if m.IsMirror {
 		return a
 	}
```

Comment:
The correct solution is to remove this entire test. 

```suggestion
```

Your solution will break pull mirrors